OSDOCS-16988-5-CQA3: CQA PR 5.1 for INSTALL-1: Core Installation Conc (#113738)
openshift/openshift-docs: modules/installation-special-config-storage.adoc

//
// * installing/install_config/installing-customizing.adoc

:_mod-docs-content-type: CONCEPT
[id="installation-special-config-storage_{context}"]
= Boot disk encryption and mirroring during installation

[role="_abstract"]

> During an {product-title} installation, you can enable boot disk encryption and mirroring on the cluster nodes.

Reviewer: Same concern as [earlier](https://github.com/openshift/openshift-docs/pull/113738/changes#r3509168265). 

```suggestion
You can configure the {product-title} installation to enable boot disk encryption and mirroring on the cluster nodes.
```

Author: :heavy_check_mark:

> You can enable encryption for the boot disks on the control plane and compute nodes at installation time. {product-title} supports the Trusted Platform Module (TPM) v2 and Tang encryption modes.

Reviewer: Is this first sentence redundant with the short desc?

```suggestion
{product-title} supports the Trusted Platform Module (TPM) v2 and Tang encryption modes.
```

Author: :heavy_check_mark:
